src/clinvar_streams/storage/database_sqlite/client.clj

```diff
@@ -2,41 +2,71 @@
   (:require [clojure.java.io :as io]
             [clojure.java.jdbc :as jdbc]
             [clojure.java.shell :refer :all]
-            [taoensso.timbre :as log])
+            [taoensso.timbre :as log]
+            [clinvar-streams.util :as util]
+            [clinvar-combiner.config :as config])
   (:import [java.io File]
            (java.sql PreparedStatement)
            (java.util Iterator)))
 
 (def db (atom {}))
 
-(def sql-root "src/clinvar_streams/storage/database_sqlite/sql")
+(def sql-resource-root "clinvar_streams/storage/database_sqlite/sql")
 
-(defn sql-path
+(defn sql-resource-path
   "Returns a cwd-rooted path for a sql filename under the project's sql directory.
   Throws exception if not found."
   [filename]
-  (.getPath (io/file sql-root filename)))
+  (str sql-resource-root "/" filename))
+
+(defn run-sql-resource [db-path filename]
+  (let [resource-path (sql-resource-path "initialize.sql")]
```

Comment:
should the SQL file name be turned into a config param? I suppose hardcoding isn't too much of a concern. just an observation.

Reply:
Resolved in f46730328eec18e50763009132ef88fcc3350795